packages/http-client/src/ceramic-http-client.ts

```diff
@@ -56,7 +56,7 @@ export default class CeramicClient implements CeramicApi {
     const keyDidResolver = KeyDidResolver.getResolver()
     const threeIdResolver = ThreeIdResolver.getResolver(this)
     this.context.resolver = new Resolver({
-      ...config.didResolver, ...threeIdResolver, ...keyDidResolver,
+      ...this._config.didResolver, ...threeIdResolver, ...keyDidResolver,
```

Comment:
The config variable should be available here.

Reply:
It's passed to the constructor

Reply:
Oh, is it if it's unset?

Reply:
yep 🙂